### PR TITLE
DM-7670: Logging framework migration 

### PR DESCRIPTION
**Pull request by hchiang2**

### CODE REVIEW

File: tests/testCoaddPsf.py [hunk 1]
```diff
@@ -32,19 +32,10 @@
...
 from lsst.afw.geom.polygon import Polygon
 import lsst.utils.tests
 
 
-try:
-    type(verbose)
-except NameError:
-    verbose = 0
-    logging.Trace.setVerbosity("meas.algorithms.Interp", verbose)
-    logging.Trace.setVerbosity("afw.detection.Psf", verbose)
```
Comment (parejkoj) [inline tests/testCoaddPsf.py]: Should we just set the verbosity of those two bits here to "INFO" or "DEBUG" and have a comment about increasing verbosity like you do with the others?

File: tests/testPsfAttributes.py [hunk 2]
```diff
@@ -36,16 +36,13 @@
...
-import lsst.pex.logging as logging
 import lsst.utils.tests
 
 try:
     type(verbose)
     import lsst.afw.display.ds9 as ds9
 except NameError:
     verbose = 0
-    logging.Trace.setVerbosity("meas.algorithms.Interp", verbose)
-    logging.Trace.setVerbosity("afw.detection.Psf", verbose)
```
Comment (parejkoj) [inline tests/testPsfAttributes.py]: I think it'd be good to do the same thing here that you do in testPsfDetermination: set logging to INFO or DEBUG or TRACE with comment on how to increase log level.

File: tests/testInterp.py [hunk 3]
```diff
@@ -33,14 +33,8 @@
...
 import lsst.meas.algorithms as algorithms
 import lsst.meas.algorithms.defects as defects
-import lsst.pex.logging as logging
 import lsst.utils.tests
 
-try:
-    type(verbose)
-except NameError:
-    verbose = 0
-    logging.Trace_setVerbosity("algorithms.Interp", verbose)
```
Comment (parejkoj) [inline tests/testInterp.py]: I think it'd be good to do the same thing here that you do in testPsfDetermination: set logging to INFO or DEBUG or TRACE with comment on how to increase log level.

File: tests/testDoubleGaussianPsf.py [hunk 4]
```diff
@@ -34,14 +34,8 @@
...
 import lsst.meas.algorithms as measAlg
 import lsst.pex.exceptions as pexExceptions
-import lsst.pex.logging as logging
 import lsst.utils.tests
 
-try:
-    type(verbose)
-except NameError:
-    verbose = 0
-    logging.Trace_setVerbosity("algorithms.Interp", verbose)
```
Comment (parejkoj) [inline tests/testDoubleGaussianPsf.py]: I think it'd be good to do the same thing here that you do in testPsfDetermination: set logging to INFO or DEBUG or TRACE with comment on how to increase log level.

File: tests/testInterp.py (hunk 3 above)
Comment (hchiang2) [inline tests/testInterp.py]: I can't find a logger called ``algorithms.Interp`` in the current master.  By running this test with the lowest log level enabled, I see a few debug messages from ``afw.image.Mask`` ("Number of mask planes: 16") but nothing from this package. So I don't know which logger is supposed to be included here.  I included notes about this in the commit message of https://github.com/lsst/meas_algorithms/commit/c33f9c998fe5757b14474aa91ccf447ead665f6f    In the future developers can add meaningful placeholders here when they add logging to the relevant codes.

Comment (hchiang2) [inline tests/testInterp.py]: I can't find a logger called ``algorithms.Interp`` in the current master.  By running this test with the lowest log level enabled, I see a few debug messages from ``afw.image.Mask`` ("Number of mask planes: 16") but nothing from this package. So I don't know which logger is supposed to be included here.  I included notes about this in the commit message of https://github.com/lsst/meas_algorithms/commit/c33f9c998fe5757b14474aa91ccf447ead665f6f    In the future developers can add meaningful placeholders here when they add logging to the relevant codes.

File: tests/testCoaddPsf.py (hunk 1 above)
Comment (hchiang2) [inline tests/testCoaddPsf.py]: Similar to ``testInterp.py``, there are debug messages from ``afw.math`` but no loggers of these names exist.

File: tests/testPsfAttributes.py (hunk 2 above)
Comment (hchiang2) [inline tests/testPsfAttributes.py]: Similar to ``testInterp.py``, there are debug messages from ``afw.image.Mask`` but no loggers of these names exist.

File: tests/testDoubleGaussianPsf.py (hunk 4 above)
Comment (hchiang2) [inline tests/testDoubleGaussianPsf.py]: Similar to ``testInterp.py``, there are debug messages from ``afw.math`` but no loggers of these names exist.

**Comment by parejkoj** [comment]
Ah! Sorry about the trouble, and thanks for following up. I should have checked the commit message.

**Pull request merged**